### PR TITLE
add icon to notification

### LINKED ISSUE #212: Add icons for notifications
Currently the notifications do not have any icon. Would simply adding an icon via img/app.svg resolve this @nickvergessen?

The icon we want here is `icon-error`, that is: https://github.com/nextcloud/server/blob/master/core/img/actions/error.svg

(Otherwise for reference, @ChristophWurst mentioned the place the notifications are generated are at https://github.com/nextcloud/suspicious_login/blob/master/lib/Notifications/Notifier.php#L66-L73 )

### PR DESCRIPTION
**Pull request by szaimen**
Fix #212

Disclaimer: I could not test this because I cannot compile the app. See https://github.com/nextcloud/suspicious_login/issues/536 

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: lib/Notifications/Notifier.php [hunk 1]
```diff
@@ -72,6 +72,7 @@ public function prepare(INotification $notification, string $languageCode): INot
 					throw new InvalidArgumentException();
 				}
 
+				$notification->setIcon($this->url->getAbsoluteURL($this->url->imagePath('suspicious_login', 'error.svg')));
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply lib/Notifications/Notifier.php:75]: Well an error icon with a 3rdparty service (#535 ) on "More information" is guiding people to click on random things.

[the remark this answers is not in the document]
Comment (szaimen) [reply lib/Notifications/Notifier.php:75]: You are probably right but how to solve this then?

[the remark this answers is not in the document]
Comment (szaimen) [reply lib/Notifications/Notifier.php:75]: I've modified the description of the button now.
See https://github.com/nextcloud/suspicious_login/commit/a46eb50f36bc91e952f9c8c62df847d4652f16a5

**Review by ChristophWurst — APPROVED** [review]

**Comment by szaimen** [comment]
Is there still something missing here?

**Comment by nickvergessen** [comment]
I think the error icon is too much. All notifications of that type I get so far were false positives, so an error icon is too scary from my pov. But I leave it up to @jancborchardt

**Comment by jancborchardt** [comment]
Sorry, for some reason the issue didn’t show in my "Review requested"!

> I think the error icon is too much. All notifications of that type I get so far were false positives, so an error icon is too scary from my pov. But I leave it up to @jancborchardt

Yeah, I also only got log ins from myself (from new places though so valid). A more generic "login" icon might indeed be more fitting: https://fonts.google.com/icons?selected=Material+Icons:login:

**Comment by szaimen** [comment]
done 👍

File: img/login.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><g><rect fill="none" height="24" width="24"/></g><g><path d="M11,7L9.6,8.4l2.6,2.6H2v2h10.2l-2.6,2.6L11,17l5-5L11,7z M20,19h-8v2h8c1.1,0,2-0.9,2-2V5c0-1.1-0.9-2-2-2h-8v2h8V19z"/></g></svg>
```
Comment (nickvergessen) [inline img/login.svg:1]: Bonus points for renaming to app.svg so it's also used as an icon for the app in app management

Comment (szaimen) [reply img/login.svg]: done

**Review by nickvergessen — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**